### PR TITLE
Schema for validatexml

### LINKED ISSUE #8: Add schema for validatexml
Add a schema for the the validatexml seen for entries in a coda site instance at APP/validate/.

### PR DESCRIPTION
**Pull request by somexpert**
This will fix #8

### CODE REVIEW

**Comment by somexpert** [comment]
@vphill, @ldko, @damonkelley, this schema, readme, and example are ready for review.

File: validatexml/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+==============
+
+
+About
+-----
+
+This schema defines what the UNT Libraries considers to be a valid validatexml
+instance. validatexml instances represent bags that are scheduled to be validated.
+Validating here refers to checking the hashes of the bags. The bags referred to
+here use the [BagIt specifications](www.digitalpreservation.gov/documents/bagitspec.pdf).
```
Comment (ldko) [inline validatexml/README.md]: I think this link needs fixing.

**Comment by damonkelley** [comment]
Is there no restriction on what the `lastVerfiedStatus` can be? Or are we not worried about that?

**Comment by damonkelley** [comment]
@ldko, what about `priority`? We only use 0 or 1 for that field, do we want to include that constraint?

**Comment by ldko** [comment]
yeah 0 or 1 would be a good constraint to add. Also, a constraint on lastVerifiedStatus would be good.

**Comment by ldko** [comment]
'Passed', 'Failed', and 'Unverified' are the possible status values.

**Comment by somexpert** [comment]
Ok, I'll add those constraints.

**Comment by somexpert** [comment]
Constraints added and link fixed.

**Pull request merged**